### PR TITLE
chore(views): get input/autocomplete working again

### PR DESCRIPTION
**Pull request by iionly**

### CODE REVIEW

**Comment by mrclay** [comment]
This close to release we should fix this in the simplest way possible. This PR would remove the `elgg.autocomplete` JS lib, breaking back compatibility.

File: views/default/input/autocomplete.php [hunk 1]
```diff
@@ -35,17 +35,11 @@
 	$params['match_owner'] = $vars['match_owner'];
 	unset($vars['match_owner']);
 }
-$ac_url_params = http_build_query($params);
+$ac_url_params = urldecode(http_build_query($params));
```
Comment (mrclay) [inline views/default/input/autocomplete.php]: Why the urldecode? This should not be necessary.

**Comment by mrclay** [comment]
On 2nd though, the implementation in 2.0 is already not BC with 1.12, so I'm OK heading in this direction, but we should document the breaking change. Working on a PR.

**Comment by mrclay** [comment]
Replaced by #9125. Thanks.

File: views/default/input/autocomplete.php (hunk 1 above)
Comment (iionly) [inline views/default/input/autocomplete.php]: @mrclay For whatever reason it didn't work without urldecode when I tested it on the weekend. For whatever reason it now also works without...

Checking out your PR now.

**Pull request closed without merging**